posthog/api/authentication.py

```diff
@@ -11,7 +15,23 @@
 from posthog.models import User
 
 
-def axess_logout(*args, **kwargs):
+@csrf_protect
+def logout(request):
+    if request.user.is_authenticated:
+        request.user.temporary_token = None
+        request.user.save()
+
+    if is_impersonated_session(request):
+        restore_original_login(request)
```

Comment:
Not sure if we should do this or completely kill the session. While this is nicer, I'm worried we're not properly handling cookies and distinct ID's when capturing data on PostHog and leading to unexpected merge users on our own team.

Reply:
As long as we call `posthog.reset()` before `.identify` we should be fine or not sure what you mean.

Reply:
You're right, we do!